### ISSUE TITLE
A few suggestions

### ISSUE THREAD

**Issue by Tiwas** [antonpup/Aurora#398]
Love the program, but wanted to ask for a few additions - but I don't know if they're all possible :)

1. Support for saving static light to mouse and keyboard, like Corsair's CUE software can.
2. Support for disabling keys when in certain applications
3. Suppress other programs' request for focus when in certain applications in order for full screen games to not minimize
4. [This might be the hardest part - and the one I'm least optimistic about] Prevent the mouse from registering outside the active application, like in my setup where I have three monitors and only game on the middle one - freaking out and trying to spray someone in CS:GO some times ends up with the game minimized and  me clicking on something in outlook or firefox...in addition to be very dead :p
5. Documentation and maybe a walk through for setting up profiles and explaining the different effects would be awesome.

(by "in certain applications", I mean either set it in a system wide config or in an application profile)

This is NOT a complaint! This is just a few ideas on how to make the application even more awesome :D

**Comment by antonpup** [comment]
> Support for saving static light to mouse and keyboard, like Corsair's CUE software can.

I am not sure what you mean, can you provide an example? Afaik you can already set the peripherals to be a static color...

> Support for disabling keys when in certain applications

It is possible to do, but current there are performance issues related to input. It is currently unknown if such feature will be possible in the future. But I have come to the conclusion that it is best for hardware to block key inputs, not software.

> Suppress other programs' request for focus when in certain applications in order for full screen games to not minimize

I don't think it's Aurora's business to suppress other applications. Its purpose is to provide unified RGB lighting.

> [This might be the hardest part - and the one I'm least optimistic about] Prevent the mouse from registering outside the active application, like in my setup where I have three monitors and only game on the middle one - freaking out and trying to spray someone in CS:GO some times ends up with the game minimized and me clicking on something in outlook or firefox...in addition to be very dead :p

Same answer as for 3, Aurora is for RGB lighting.

> Documentation and maybe a walk through for setting up profiles and explaining the different effects would be awesome.

Yes, I can agree that there is a lack of tutorials regarding Aurora. I am hoping that someone from community would start creating guides, but I can do some tutorial videos to start off.

**Comment by Tiwas** [comment]
>> Support for saving static light to mouse and keyboard, like Corsair's CUE software can.
> I am not sure what you mean, can you provide an example? Afaik you can already set the peripherals to be a static color...
Yes, but CUE can upload it to the device so that a certain static light is active when the software isn't. This is mostly for starting windows or when the computer has shut down (if you're using a powered usb hub). No big deal as I can temporarily stop Aurora, get CUE to upload it to the device's memory and then restart Aurora.

As for #2: ok :)

As for #3 and #4: That's what I thought - but figured I could ask just to make sure :D

Video tutorials would be super cool!

And again - awesome piece of software. Love it! <3

**Comment by GuyMan1020** [comment]
> Yes, I can agree that there is a lack of tutorials regarding Aurora. I am hoping that someone from community would start creating guides, but I can do some tutorial videos to start off.

One way to try and get people to make some tutorials is to create a steam group, I'm still trying to look for a good tutorial on making scripts for aurora
